kubecontrolplane/v1/types.go

```diff
@@ -54,6 +54,20 @@ type KubeAPIServerConfig struct {
 	// If no keys are specified, no service account authentication will be available.
 	ServiceAccountPublicKeyFiles []string `json:"serviceAccountPublicKeyFiles"`
 
+	// serviceAccountIssuer is the identifier of the bound service account token
+	// issuer. If not provided, a default value of "auth.openshift.io" will be used.
+	ServiceAccountIssuer string `json:"serviceAccountIssuer"`
```

Comment:
these can be passed as flags, right? If we can use flags, then we don't add the types here.  We want to eliminate this type over time and only use the upstream provided values.

Reply:
That sound preferable, but it is not clear from reading the code how exactly these flags would be specified by the operator (or overridden by an administrator). Are there docs/examples/code/etc you can point me to?

Reply:
Done